src/CentralBank.java

```diff
@@ -0,0 +1,128 @@
+import java.math.BigDecimal;
+import java.util.*;
+
+public class CentralBank
+{
+    private List<Account> accounts;
+    private Map<String, InterestCalculator> Calcualateinterest;
```

Comment:
변수명은 카멜케이스 표기법을 사용합시다!
```suggestion
    private Map<String, InterestCalculator> calcualateInterest;
```